runtime/browser/runtime.cc

```diff
@@ -225,13 +222,10 @@ void Runtime::DidNavigateMainFramePostCommit(
 
 content::JavaScriptDialogManager* Runtime::GetJavaScriptDialogManager(
     content::WebContents* web_contents) {
-#if defined(OS_LINUX)
+#if defined(USE_AURA)
```

Comment:
The guard defined(USE_AURA) is a little bit confused, actually app_modal::JavaScriptDialogManager is platform independent. It also applies to Android.